### DISCUSSION TITLE
Sorting out my coding

### DISCUSSION THREAD

**Discussion by BakariMtili** [IBAHCM/RPiR#170] [Archive] [answered]
Hello @soniamitchell I am getting this error message '  Function step_deterministic_SIS() may not use global variable(s): infectedlatest.infectedssusceptibles' what do you think might be the problem?
thank you

**Comment by BakariMtili** [comment, marked as answer]
Hello @ I have already sorted the problem! thanks

**Reply by richardreeve** [reply]
Great! Happy to have it fixed before we get to it :)
